.github/workflows/nightly-check-db.yml

```diff
@@ -0,0 +1,40 @@
+name: Nightly check db
+
+on: [push]
+  # schedule:
+  #   - cron: '0 3 * * *'
+
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
+jobs:
+  download-snapshaot-move-data-check-db:
+
+    runs-on: ubuntu-latest
+    container: quarkchaindocker/pyquarkchain:mainnet1.1.1
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Download db snapshot
+      env:
+        PYTHONPATH: /tmp
```

Comment:
hmm I don't think PYTHON is needed here. do you mean current working directory?

Reply:
yes, I have used the keyword of working directory instead.